samcli/lib/test_runner/fargate_testsuite_runner.py

```diff
@@ -68,8 +64,7 @@ def __init__(
         self.deployer = Deployer(cloudformation_client=self.boto_cloudformation_client)
 
     def _read_file(self, filename: str) -> str:
```

Comment:
Two questions; do you need to keep filename as a string, or can this be `file: Path`?
If it could be a path, do you need this helper method or would it be better to call `.read_text()` directly?

I apologize if I missed a place where _read_file was called with a plain string

Reply:
Yeah, I left the _read_file wrapper just so I could keep the test mocks, but it's definitely worth reducing the number of functions by just taking it out. I'll remove it and update the tests :)